packages/renderer/src/stores/create-network.ts

```diff
@@ -0,0 +1,57 @@
+/**********************************************************************
```

Comment:
Why is this file in the `store` folder ? This should not be here unless it is a store